html-tables/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html tables</title>
+  </head>
+  <body>
+    <h1><strong>Student Grade Table</strong></h1>
+    <table>
+      <thead>
+        <tr>
+          <th><strong>Studentname</strong></th>
+          <th><strong>Course</strong></th>
+          <th><strong>Grade</strong></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Grace Hopper</td>
+          <td>Computer Science</td>
+          <td>100</td>
+        </tr>
+        <tr>
+          <td>Alan Turning</td>
+          <td>Mathematics</td>
+          <td>100</td>
+        </tr>
+        <tr>
+          <td>Douglas Crockford</td>
+          <td>JSON</td>
+          <td>98</td>
+        </tr>
+        <tr>
+          <td>Tim Berners-Lee</td>
+          <td>HTML</td>
+          <td>9001</td>
+        </tr>
+      </tbody>
+      <tfoot>
+        <tr>
+          <td>Linus Torvalds</td>
+          <td>Git</td>
+          <td>90</td>
+        </tr>
+      </tfoot>
```

Comment:
This is technically still apart of your body, so it should be inside the `tbody` rather than the `tfoot`, but I understand your thought process